FantasmoSDK/Classes/FMLocationManager.swift

```diff
@@ -118,7 +118,7 @@ open class FMLocationManager: NSObject, FMApiDelegate {
     private var lastCLLocation: CLLocation?
     private weak var delegate: FMLocationDelegate?
     
-    private var arKitTrackingStateStatistics = ARKitTrackingStateStatistics()
+    private var accumulatedARKitRelatedInfo = AccumulatedARKitRelatedInfo()
```

Comment:
How about just `accumulatedARKitInfo`? The `related` part seems superfluous.

Reply:
Just thought that simply "ARKit info" is perceived as info about ARKit itself. But now I think it will be appropriate name. Done